modules/oxide/src/less/theme/components/button/button-icon.less

```diff
@@ -13,6 +13,16 @@
       display: block;
       fill: currentColor;
     }
+
```

Comment:
I would avoid setting a general variable as the value, as these are (unfortunately) too generic to fit all scenarios, so I recommend creating component-specific variables that allows for customizations if necessary. Something like `@icon-success-color: @color-success`.

Reply:
Just one would do fine I think